src/madetomeasure_api/routes/subscribers.clj

```diff
@@ -29,8 +29,8 @@
 (defn create-subscribers [request]
   (let [subscribers (extract-subscribers request)]
     (try
-      (apply clojure.java.jdbc/insert! db/db-spec :subscribers subscribers)
-      (present-response/success subscribers)
+      (let [insertions (apply clojure.java.jdbc/insert! db/db-spec :subscribers subscribers)]
```

Comment:
Didn't realize this but the response wasn't returning the response from the db.

This will return what gets inserted including the id / updated_at etc.